server/src/test/java/com/cloud/server/StatsCollectorTest.java

```diff
@@ -511,4 +512,16 @@ public void testPersistVolumeStatsKVM() {
     public void testPersistVolumeStatsVmware() {
         performPersistVolumeStatsTest(Hypervisor.HypervisorType.VMware);
     }
+
+    private Map<String, String> convertJsonToOrderedMap(String json) {
+        Map<String, String> jsonMap = new TreeMap<String, String>();
+        String[] keyValuePairs = json.replace("{", "").replace("}","").split(",");
+        for(String pair: keyValuePairs) {
+            String[] keyValue = pair.split(":");
+            String key = keyValue[0];
+            String value = keyValue[1];
+            jsonMap.put(key, value);
```

Comment:
Consider reducing the number of lines by not creating new variables `key` and `value` as they are used only once. Use `jsonMap.put(keyValue[0], keyValue[1])`

Reply:
Thanks, addressed